src/components/stores.js

```diff
@@ -1,19 +1,19 @@
-import { writable } from 'svelte/store';
```

Comment:
I'm thinking this file could be removed...

Reply:
Agreed.